test/e2e.env

```diff
@@ -32,28 +41,37 @@ if [[ "$(which chromium)" =~ "/snap/bin/" ]]; then
   export E2E_TMPDIR=./tmp;
 fi;
 
+export E2E_DATA_DIR=./test-data
+export E2E_CHROME_USER_DATA=test-data/chrome-user-data
+# export E2E_SETTINGS_ARCHIVE=test-data/settings.tar.gz
+export E2E_SETTINGS_ARCHIVE_URL="https://github.com/mlabs-haskell/ctl-e2e-assets/releases/download/Latest/settings.tar.gz"
+
+
 export E2E_TEST_TIMEOUT=200
 export E2E_BROWSER=chromium
 export E2E_NO_HEADLESS=false
-export E2E_SETTINGS_ARCHIVE=test-data/settings.tar.gz
-export E2E_CHROME_USER_DATA="test-data/chrome-user-data"
 
-export ETERNL_CRX="test-data/chrome-extensions/eternl_1.9.1_0.crx"
+# export ETERNL_CRX="test-data/chrome-extensions/eternl_1.9.1_0.crx"
+export ETERNL_CRX_URL="https://github.com/mlabs-haskell/ctl-e2e-assets/releases/download/Latest/eternl_1.9.1_0.crx"
 export ETERNL_PASSWORD=ctlctlctlctl
 export ETERNL_EXTID=eafgjihkcgemnnhmidkfinncmncolppd
 
-export LODE_CRX="test-data/chrome-extensions/lode_1.0.1.1_0.crx"
+# export LODE_CRX="test-data/chrome-extensions/lode_1.0.1.1_0.crx"
+export LODE_CRX_URL="https://github.com/mlabs-haskell/ctl-e2e-assets/releases/download/Latest/lode_1.0.1.1_0.crx"
 export LODE_PASSWORD=ctlctlctl
 export LODE_EXTID=glajohhncdfhgdjglmfnoijpdibocada
 
-# export NAMI_CRX="test-data/chrome-extensions/nami_3.2.5_1.crx"
-# export NAMI_PASSWORD=ctlctlctl
-# export NAMI_EXTID=lpfcbjknijpeeillifnkikgncikgfhdo
+# export NAMI_CRX="test-data/chrome-extensions/nami_3.4.1_0.crx"
+export NAMI_CRX_URL="https://github.com/mlabs-haskell/ctl-e2e-assets/releases/download/Latest/nami_3.4.1_0.crx"
+export NAMI_PASSWORD=ctlctlctl
+export NAMI_EXTID=lpfcbjknijpeeillifnkikgncikgfhdo
 
 # export FLINT_CRX="test-data/chrome-extensions/flint_1.16.2_0.crx"
-# export FLINT_PASSWORD=VZVfu5rp1r
-# export FLINT_EXTID=hnhobjmcibchnmglfbldbfabcgaknlkj
+export FLINT_CRX_URL="https://github.com/mlabs-haskell/ctl-e2e-assets/releases/download/Latest/flint_1.16.2_0.crx"
```

Comment:
We can't identify the files by URLs alone. We need to allow to pass files.

If the file is not present, then we should use the URL to download the remote file to the file location.

Otherwise, just use the file. We should make the URLs optional, but not the files.